kv/txn_test.go

```diff
@@ -393,9 +393,9 @@ func TestUncertaintyMaxTimestampForwarding(t *testing.T) {
 		// node clock (which is ahead of keyFast as well). If the last part does
 		// not happen, the read of keyFast should fail (i.e. read nothing).
 		// There will be exactly one restart here.
-		call = client.Get(keySlow)
-		gr := call.Reply.(*proto.GetResponse)
-		if err = txn.Run(call); err != nil {
+		getCall := client.Get(keySlow)
+		gr := getCall.Reply
```

Comment:
feels kind of weird to pull out the `.Reply` before it's being populated. Sure, it's correct, but it requires you to know that `client.Get()` pre-initializes the `Reply` and that `Run` never replaces it.